src/tools/interop/interop.cpp

```diff
@@ -795,6 +923,9 @@ RunInteropTest(
                 !Connection.GetResumptionTicket(ResumptionTicket, ResumptionTicketLength)) {
                 break;
             }
+            if (SslKeyLogFile) {
+                Connection.WriteSslKeyLogFile(SslKeyLogFile);
+            }
```

Comment:
There's got to be a better way instead of doing this instead of copying this all over the place. Maybe pass the `SslKeyLogFile` to the `Connection` and on cleanup if it's set, write the file?

Though, that brings up another point. What synchronization are you doing around the file. If you open a lot of connection in parallel, they'd all be writing to the file from multiple parallel threads, right? That would result in a big jumble.